docs/usage/self-hosting.md

```diff
@@ -201,6 +201,13 @@ Self-hosted Renovate can be configured using any of the following (or a combinat
 Note that some Renovate configuration options are _only_ available for self-hosting, and so can only be configured using one of the above methods.
 These are described in the [Self-hosted Configuration](./self-hosted-configuration.md) doc.
 
+If you are configuring using env variables, there are two possibilities:
```

Comment:
We use `Environment parameters` (Line 199) and `Environment variables` (Line 204).
It's better if we pick _one_ wording and use it throughout this document.
Whatever wording we choose, we should spell it out in full instead of abbreviating it (`env` -> `environment`).

I'll let you decide what you like best.

Reply:
Both should be "variables" - resolved